### DISCUSSION TITLE
State "Heating"

### DISCUSSION THREAD

**Discussion by RealGue** [mtrossbach/noah-mqtt#31] [Ideas]
Hi, 

last week I discovered a state "Heating" in the ShinePhone-App for the 1st time
![Screenshot_20241015_084445_ShinePhone](https://github.com/user-attachments/assets/e0b0e662-d955-4781-bec6-bc776b79b88d)

HA still shows "Disconnected".
It would be nice if the "Heating"-state would visible in HA.

Many thanks!
Jörn

**Comment by msmthng** [comment]
yes would be nice

**Comment by msmthng** [comment]
i have added the heating status and created a pull request https://github.com/mtrossbach/noah-mqtt/pull/55

**Reply by les-les** [reply]
Thank you looks like gonna be in the next Release  0.0.26 

Will be awesome

**Comment by les-les** [comment]
Works like a Charme in 0.0.26
